bucket/aider-desk.json

```
@@ -0,0 +1,40 @@
+{
+    "version": "0.3.1",
+    "description": "Desktop application for Aider AI assistant",
+    "homepage": "https://github.com/hotovo/aider-desk/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hotovo/aider-desk/releases/download/v0.3.1/aider-desk-0.3.1-setup.exe#/dl.7z",
+            "hash": "sha512:52a8f0dcaa26d4abb2921f83513e5fadf14ec2a21b6df96e31eeee43e48665fdde4d0cfe88d3a76470186ec4e664cba85dc4dca19b76e3d6ed23ef7dc295cf2b",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" -DestinationPath \"$dir\"",
+                "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Recurse"
+            ],
+            "post_uninstall": [
+                "Remove-Item \"$env:APPDATA\\aider-desk\" -Recurse -Force"
+            ]
```

```suggestion
            "post_uninstall": [
                "Remove-Item \"$env:APPDATA\\aider-desk\" -Recurse -Force"
            ]
```

Guess it would be nice not simply delete `AppData` entry but manually persist user settings (if any), see e.g. [cherry-studio](https://github.com/ScoopInstaller/Extras/blob/master/bucket/cherry-studio.json).

---

thanks for suggestions